superset/views/core.py

```diff
@@ -298,6 +304,94 @@ class DatabaseAsync(DatabaseView):
 appbuilder.add_view_no_menu(DatabaseAsync)
 
 
+@app.route('/uploads/<filename>')
+def uploaded_file(filename):
+    with open("flask-stream-demo", "bw") as f:
+        chunk_size = 4096
+        while True:
+            chunk = flask.request.stream.read(chunk_size).decode('utf-8')
+            if len(chunk) == 0:
+                return
+            f.write(chunk)
+
+class CsvToDatabaseView(SimpleFormView):
+    #DatasourceModelView,  add to ^ ?
+    form = CsvToDatabaseForm
+    form_title = _('CSV to Database configuration')
+    add_columns = ['database', 'schema', 'table_name']
+
+    def form_get(self, form):
+        # pre process form
+        # default values
+        form.name.data = None
```

Comment:
Also lots of boilerplate here.

Reply:
These set the default values. I removed the ones that weren't changing anything.